src/routers/kbRouters.js

```diff
@@ -0,0 +1,73 @@
+import express from 'express'
+import { App } from '../types/App'
+import { Settings } from '../types/Settings'
+
+const { KB_CONFIG } = require('../environment')
+const { KubeConfig } = require('kubernetes-client')
+const Request = require('kubernetes-client/backends/request')
+const Client = require('kubernetes-client').Client
+
+const router = express.Router()
+
+router.get('/list/namespaces', async (req, res) => {
+  try {
+    const kubeconfig = new KubeConfig()
+    console.log('kb config', KB_CONFIG)
+    // kubeconfig.loadFromString(JSON.stringify(config))
+    kubeconfig.loadFromFile(KB_CONFIG)
+
+    const backend = new Request({ kubeconfig })
+    const client = new Client({ backend, version: '1.13' })
+    const namespaces = await client.api.v1.namespaces.get()
+    console.log('namespace', namespaces)
+    res.status(200).json({
+      status: 'OK',
+      result: namespaces
+    })
+  } catch (err) {
+    console.log('error', err)
+  }
+})
+
+router.post('/deploy/promster', async (req, res) => {
+  try {
+    const kubeconfig = new KubeConfig()
+    kubeconfig.loadFromFile(KB_CONFIG)
+
+    // kubeconfig.loadFromString(JSON.stringify(config))
```

Comment:
@gutorc92, you can move this logic outside this code. It is a duplicate of lines 14-17